lightning/src/ln/peers/chacha.rs

```diff
@@ -0,0 +1,34 @@
+use util::chacha20poly1305rfc::ChaCha20Poly1305RFC;
+
```

Comment:
Why is this in ln::peers? It seems to be pure crypto functions.

Reply:
Because the AEAD-based encryption methods are only used for handshakes and peer message encryption IIRC, and not for the onion construction.

Reply:
Right, but its also a pure-crypto primitive. I guess my preference is for such things (even if it implements a lightning protocol crypto primitive) to be in some kind of crypto module.